component/attr_cache/attr_cache_test.go

```diff
@@ -210,8 +207,7 @@ func (suite *attrCacheTestSuite) TestDefault() {
 	defer suite.cleanupTest()
 	suite.assert.Equal(suite.attrCache.Name(), "attr_cache")
 	suite.assert.EqualValues(suite.attrCache.cacheTimeout, 120)
-	suite.assert.Equal(suite.attrCache.cacheOnList, true)
-	suite.assert.Equal(suite.attrCache.noSymlinks, false)
+	// suite.assert.Equal(suite.attrCache.noSymlinks, false)
```

Comment:
I think this is not needed. Please remove it.

Reply:
this is to remove an older feature which is not implemented correctly. We have symlink related flags being handled at both attr-cache and azstorage and for some reason we had different defualt behaviour like azstorage assume symlink is on by default while attr-cache assume its off. So as part of this PR I am removing that duplicate flag from attr-cache.